pkg/ui/src/views/cluster/containers/timescale/timescale.spec.tsx

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+import React from "react";
+// import { assert } from "chai";
+import { mount } from "enzyme";
+import _ from "lodash";
+import {
+  TimeScaleDropdown,
+  TimeScaleDropdownProps,
+  getTimeRangeTitle,
+  generateDisabledArrows,
+} from "./index";
+import * as timewindow from "src/redux/timewindow";
+// import sinon from "sinon";
+import moment from "moment";
+import { refreshNodes } from "src/redux/apiReducers";
+import "src/enzymeInit";
+import { MemoryRouter } from "react-router";
+import TimeFrameControls from "../../components/controls";
+import RangeSelect from "../../components/range";
+import { expect, assert } from "chai";
+import sinon from "sinon";
+import { ArrowDirection } from "oss/src/views/shared/components/dropdown";
+
+const initialEntries = [
+  "#/metrics/overview/cluster", // Past 10 minutes
+  `#/metrics/overview/cluster/cluster?start=${moment()
+    .subtract(1, "hour")
+    .format("X")}&end=${moment().format("X")}`, // Past 1 hour
+  `#/metrics/overview/cluster/cluster?start=${moment()
+    .subtract(6, "hours")
+    .format("X")}&end=${moment().format("X")}`, // Past 6 hours
+  "#/metrics/overview/cluster/cluster?start=1584528492&end=1584529092", // 10 minutes
+  "#/metrics/overview/cluster?start=1583319565&end=1584529165", // 2 weeks
+  "#/metrics/overview/node/1", // Node 1 - Past 10 minutes
+  `#/metrics/overview/node/2?start=${moment()
+    .subtract(10, "minutes")
+    .format("X")}&end=${moment().format("X")}`, // Node 2 - Past 10 minutes
+  "#/metrics/overview/node/3?start=1584528726&end=1584529326", // Node 3 - 10 minutes
+];
+
+const dateFormat = "MMM DD,";
```

Comment:
Would it be better to reuse `dateFormat` and `timeFormat` by exporting them and importing them in tests? Will reduce code duplication.